setup/sale_order_line_serial_unique/.eggs/README.txt

```diff
@@ -0,0 +1,6 @@
+This directory contains eggs that were downloaded by setuptools to build, test, and run plug-ins.
+
+This directory caches those eggs to prevent repeated downloads.
+
+However, it is safe to delete this directory.
+
```

Comment:
Do not commit these .eggs directory, please. I recommend putting `.eggs/` in your global personal .gitignore (along with your IDE files etc).